MICADO/test_micado/test_micado_compiles.py

```diff
@@ -57,7 +57,7 @@ def test_user_commands_loads_mode_files(self):
 
     def test_user_commands_can_change_modes(self):
         cmd = scopesim.UserCommands(use_instrument="MICADO")
-        cmd.set_modes(["MCAO", "SPEC_3000x50"])
+        cmd.set_modes("MCAO", "SPEC_3000x50")
```

Comment:
Did you change this in the opposite direction? It seems the tests fail on this part
```suggestion
        cmd.set_modes(["MCAO", "SPEC_3000x50"])
```